lighthouse-core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -427,7 +464,8 @@ describe('Trace Elements gatherer - GetTopLayoutShiftElements', () => {
         },
       ])
     );
-    trace.traceEvents.push(makeAnimationTraceEvent('1', 5));
+    trace.traceEvents.push(makeAnimationTraceEvent('0x363db876c8', 'b', {id: '1', nodeId: 5}));
```

Comment:
would be nice to try to run this through a real trace too, did you end up checking one in already or was it removed through the stages of review here? I seemed to remember you having one

Reply:
The test trace was originally used in test cases for the audit, but after moving all the trace logic to the gatherer I removed it since it was no longer being used.  It's easy to add back, so I'll do that.

Reply:
bump :)